extensions-api/src/main/scala/pl/touk/nussknacker/engine/testmode/TestProcess.scala

```diff
@@ -33,15 +33,8 @@ object TestProcess {
       )
     }
 
-    def updateExceptionResult(espExceptionInfo: NuExceptionInfo[_ <: Throwable]) = {
-      copy(exceptions =
-        exceptions :+ ExceptionResult(
-          toResult(espExceptionInfo.context),
-          espExceptionInfo.nodeComponentInfo.map(_.nodeId),
-          espExceptionInfo.throwable
-        )
-      )
-    }
+    def updateExceptionResult(espExceptionInfo: NuExceptionInfo[_ <: Throwable]) =
```

Comment:
Do we need this method? It is just a `copy` now

Reply:
I will leave it. After the change there are some setbacks with type:
![image](https://github.com/TouK/nussknacker/assets/907665/265c15f6-bca3-4f46-8664-5c95aa4c3f1f)
The effort to fix it doesn't have sens.